### PR TITLE
Fixed Typos

### PR DESCRIPTION
**Pull request by Biswajee**
Fixed minor typing errors.

### CODE REVIEW

File: modules/exploits/windows/smb/ms17_010_eternalblue.rb [hunk 1]
```diff
@@ -678,7 +678,7 @@ def smb_pass
   end
 
   # Returns the value to be passed to SMB clients for
-  # the username. If the user hs not supplied a username
+  # the username. If the user has not supplied an username
```
Comment (jrobles-r7) [inline modules/exploits/windows/smb/ms17_010_eternalblue.rb]: `a username` would be the [usage](https://www.grammar.com/a-vs-an-when-to-use/). Fix it and I'll land the PR.

**Comment by Biswajee** [comment]
Yes, I have the made the necessary change.

File: modules/exploits/windows/smb/ms17_010_eternalblue.rb (hunk 1 above)
Comment (Biswajee) [reply modules/exploits/windows/smb/ms17_010_eternalblue.rb]: I have made the changes.
Thanks.

**Comment by h00die** [comment]
# Release Notes
This PR fixes a few spelling errors in ms17-10 comments.

**Comment by h00die** [comment]
@Biswajee thanks for the submission!

**Pull request merged**